validphys2/src/validphys/photon_pdf/compute_photon.py

```diff
@@ -0,0 +1,192 @@
+"""Script that calls fiatlux to add the photon PDF."""
+
+# from validphys import lhapdfset
+from validphys.api import API
+import lhapdf
+import fiatlux
+import numpy as np
+from eko.couplings import Couplings
+from eko.compatibility import update_theory
+from eko.runner import Runner
+from .structure_functions import StructureFunction
+
+import yaml
+
+class Photon:
```

Comment:
You can see that here the class is used just to create a scope.
But since it is spanning the whole module, you already have the module scope (the other distinctive feature of carrying a state is not used, because of the comment above, i.e. the object survives one line).

Of course, I'm not suggesting to use module level variables (it would create a singleton, and globals should be always discouraged), but you can have the current `__init__` function returning a dictionary, and this will be used by the `compute` function, that would be the equivalent of what is now in `model_trainer.py`:

```py
def compute(...):
    configs = init(...)
    return photon_fitting_scale(configs, xgrids)
```